### PR TITLE
ML-RHDEVDOCS-2857: Added a step to verify that all components were installed

### PR DESCRIPTION
**Pull request by LunguTheEngineer**
Purpose: To resolve this issue: https://issues.redhat.com/browse/RHDEVDOCS-2857

Aligned team: DevTools

OCP versions this PR applies to: enterprise 4.9 and later

Content for preview: https://58045--docspreview.netlify.app/openshift-enterprise/latest/cicd/pipelines/installing-pipelines.html

SME review: @vdemeester

QE review: @VeereshAradhya 

Peer review: @rh-tokeefe

### CODE REVIEW

**Comment by masauso-lungu** [comment]
Helllo @vdemeester,
This is ready for SME review.

Regards,
masauso

**Review by vdemeester — APPROVED** [review]
LGTM 🐯

**Comment by masauso-lungu** [comment]
Hello @VeereshAradhya ,

This is ready for QE review.

Best regards,

Masauso

File: modules/op-installing-pipelines-operator-in-web-console.adoc [hunk 1]
```diff
@@ -57,3 +57,30 @@ The Operator is installed automatically into the `openshift-operators` namespace
...
++
+. Verify the installation of all components.
++
+To verify that all components of the {pipelines-title} Operator were installed successfully, login to your cluster on the terminal and run the following commands:
++
+----
+$ oc get tektonconfig config
+NAME     VERSION   READY   REASON
+config   1.9.2     True    
+$ oc get tektonpipeline,tektontrigger,tektonaddon
```
Comment (VeereshAradhya) [inline modules/op-installing-pipelines-operator-in-web-console.adoc:74]: Maybe we can add `pac` as well. `oc get tektonpipeline,tektontrigger,tektonaddon,pac`

Comment (masauso-lungu) [reply modules/op-installing-pipelines-operator-in-web-console.adoc]: Done !

File: modules/op-installing-pipelines-operator-in-web-console.adoc [hunk 2]
```diff
@@ -57,3 +57,33 @@ The Operator is installed automatically into the `openshift-operators` namespace
 ====
 +
 . Verify that the *Status* is set to *Succeeded Up to date*  to confirm successful installation of {pipelines-title} Operator.
++
+[WARNING]
+====
+The success status may show as *Succeeded Up to date* even if installation of other components is in-progress. Therefore, it is important to verify the installation manually in the terminal.
+====
```
Comment (rh-tokeefe) [inline modules/op-installing-pipelines-operator-in-web-console.adoc:64]: I think you can remove the text on line 66 and just have the instruction for step 7 state: "Verify that all components of the Red Hat OpenShift Pipelines Operator were installed successfully. Login to the cluster on the terminal, and run the following commands:"

File: modules/op-installing-pipelines-operator-in-web-console.adoc [hunk 3]
```diff
@@ -57,3 +57,31 @@ The Operator is installed automatically into the `openshift-operators` namespace
...
+$ oc get tektonpipeline,tektontrigger,tektonaddon,pac
+NAME                                          VERSION   READY   REASON
+tektonpipeline.operator.tekton.dev/pipeline   v0.41.1   True 
+NAME                                        VERSION   READY   REASON
+tektontrigger.operator.tekton.dev/trigger   v0.22.2   True    
+NAME                                    VERSION   READY   REASON
+tektonaddon.operator.tekton.dev/addon   1.9.2     True 
+NAME                                                             VERSION   READY   REASON
+openshiftpipelinesascode.operator.tekton.dev/pipelines-as-code   v0.15.5   True 
+----
```
Comment (abrennan89) [inline modules/op-installing-pipelines-operator-in-web-console.adoc:82]: Please revise the formatting here per the OCP docs guidelines:
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output

Each command and output should be a separate code block

**Review by abrennan89 — APPROVED** [review]
lgtm

**Pull request merged**